scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1999,7 +2000,10 @@ class FormatOps(
     // if  match  return  then  throw  try  while  yield
     def unapply(
         ftMeta: FormatToken.Meta
-    )(implicit style: ScalafmtConfig): Option[Seq[Split]] = {
+    )(implicit
+        style: ScalafmtConfig,
+        line: sourcecode.Line
```

Comment:
this line is always the same, right? it's not useful for debugging.

Reply:
Removed